### PR TITLE
Skinning (again⁶)

### PR DESCRIPTION
**Pull request by MegaApplePi**
## Description

`Skinning` and `Skinning/Tutorial` and `Skinning/skin.ini` and `Skinning/Tutorial/skin.ini`

## Status

~~Pending Review~~ Done

## Changelog

- Skinning: rip out some FAQ things here and move to Tutorial
  - and add subpage articles
  - and undo single lines and use paragraphs
- Skinning/Tutorial: paste ripped parts here
  - and format it
  - and main page link back to skinning
  - and use paragraph instead of single lines
- Skinning: add related links
-  Skinning/Tutorial: add related links
- skin.ini: many fixes (see description)
  - full stops and sentence case the versioning changes for bullet points
  - replace "So not" with "\*\*Never\*\*"
  - remove FAQ, to be moved to Skinning/Tutorial/skin.ini
- skin.ini: put back FAQ but call it Notes
  - just realized that it is most talking about the tables below
- Skinning/Tutorial/skin.ini: port the guide over
  - this one <https://osu.ppy.sh/community/forums/topics/575880>

## Related

#387

### CODE REVIEW

File: wiki/Skinning/Tutorial/en.md [hunk 1]
```diff
@@ -1,105 +1,136 @@
...
+- [Skinning/Tutorial/osu!taiko](/wiki/Skinning/Tutorial/osu!taiko)
+- [Skinning/Tutorial/osu!catch](/wiki/Skinning/Tutorial/osu!catch)
+- [Skinning/Tutorial/osu!mania](/wiki/Skinning/Tutorial/osu!mania)
+- [Skinning/Tutorial/Sounds](/wiki/Skinning/Tutorial/Sounds)
+
 ## Skinning FAQ
 
+### What format should the images be in?
+
+Images should **always** be in the `.png` format. `menu-background.jpg` is the expection, this one can use `.jpg`.
```
Comment (TPGPL) [inline wiki/Skinning/Tutorial/en.md]: is an exception?

File: wiki/Skinning/Tutorial/en.md [hunk 2]
```diff
@@ -1,105 +1,136 @@
...
+- **Compress** images whenever possible. Compressing usually removes unnecessary info about blank pixels and reduces the file sizes dramatically.
+- Understand that certain elements **face a certain direction.** Some elements have a standard as to which direction they must face to face the correct direction when the osu!client uses it.
+
+### What are overlays?
+
+Overlays will *overlay* an element's image **and will not be colored or tinted**. Note that only a few elements contains an overlay.
+
+### Resolutions
+
+Because _osu!_ can be played at different resolutions, some skinning elements may overlap or be farther apart than what is expected. Meaning that not elements will scale themselves to fit the game window resolution.
```
Comment (TPGPL) [inline wiki/Skinning/Tutorial/en.md]: starting a sentence with a conjuction?

Meaning -> That means

File: wiki/Skinning/Tutorial/en.md [hunk 3]
```diff
@@ -1,105 +1,136 @@
...
+- 2048 x 1536 (4:3, standard game resolution in HD scaling)
+- 1366 x 768 (16:9, standard game widescreen resolution)
+- 2732 x 1536 (16:9, standard game widescreen resolution in HD scaling)
+- 1920 x 1080 (16:9, standard HD resolution)
+
+Images will be adjusted by the game itself to fit resolutions derivating from the ones mentioned above. Most of them will be rescaled to fit the playfield or repositioned on different aspect ratios.
+
+#### HD Sprites
+
+HD sprites are used in place of the normal sprites on higher resolutions, when present. Compared to normal sprites, HD sprites look much sharper and cleaner on these larger resolutions. They get scaled by the game itself automatically to fit the resolution used. Resolutions supporting HD sprites begin at a **minimum of 800 pixel in height**. HD sprites are tagged with `@2x` at the end of their names. For example: `cursor@2x.png`.
```
Comment (TPGPL) [inline wiki/Skinning/Tutorial/en.md]: 800 pixel -> 800 pixels ?

File: wiki/Skinning/Tutorial/en.md [hunk 4]
```diff
@@ -1,105 +1,136 @@
...
+HD sprites have doubled dimension sizes. For example: the normal `hitcircle.png` sprite has the dimension sizes of **128x128 pixels.** Where its HD sprite, `hitcircle@2x.png`, has the dimension sizes of **256x256 pixels.**
+
+Every sprite has an HD counterpart, even all frames in an animation can have HD counterparts. As a result, the filesize of the folder or archive will increase due to having more images than normal.
+
+All HD sprites may also be bigger in filesize due to the fact that the canvas size used is four times bigger compared to the normal sprite. There are essentially two resolution modes _osu!_ is using. Each of them prefers one set of sprites. The first mode is **LowResolution** while the second mode is **HighResolution.**
+
+- *LowResolution mode* uses the normal sprites and ignores the HD sprites **(SD-resolution skin)**
+- *HighResolution mode* prefers HD sprites and uses normal sprites as a fallback if no HD sprite is available **(HD-resolution skin)**
+
 ### Can someone make this skin from that show/game?
```
Comment (TPGPL) [inline wiki/Skinning/Tutorial/en.md]: Can someone make a skin about that universe?

File: wiki/Skinning/Tutorial/en.md [hunk 5]
```diff
@@ -1,105 +1,136 @@
...
 
-Nonetheless, please, **never** request for a skin anywhere in the osu!fourms.
-Failing to follow this rule will your thread be moved to the wastelands.
+Nonetheless, please, **never** request for a skin anywhere in the osu!forums. Failing to follow this rule will your thread be moved to the wastelands.
 
 ### Where can I get this skin that a YouTuber/streamer played with?
 
-First off, don't go asking this in the osu!forums, your thread will most likely be thrown into the wastelands.
-Secondly, your best bet is to simply ask that person for the skin they're using.
+First off, don't go asking this in the osu!forums, your thread will most likely be thrown into the wastelands. Secondly, your best bet is to simply ask that person for the skin they're using.
```
Comment (TPGPL) [inline wiki/Skinning/Tutorial/en.md]: dont go asking -> dont ask
your best bet -> the best way

File: wiki/Skinning/Tutorial/en.md [hunk 6]
```diff
@@ -1,105 +1,136 @@
...
 
 ### I want to make a skin! But what do I need?
 
-Well nice choice, but remember, skinning can be little hard for a newbie.
-In the beginning of making any skin, you should have:
+Well nice choice, but remember, skinning can be little hard for a newbie. In the beginning of making any skin, you should have:
```
Comment (TPGPL) [inline wiki/Skinning/Tutorial/en.md]: for a newbie -> for newbies

In the beginning of making any skin -> Firstly

File: wiki/Skinning/Tutorial/en.md [hunk 7]
```diff
@@ -1,105 +1,136 @@
...
 
-- beginner skills with any photo editing program (that supports transparency) like Photoshop, GIMP, Paint.NET, etc.
-- patience (yes, skinning does take some time)
-- an idea! (yes, this is **very** important. Try be clever when making your skin)
+- Beginner skills with any photo editing program (that supports transparency) like Photoshop, GIMP, Paint.NET, etc.
+- Patience (yes, skinning does take some time)
+- An idea! (yes, this is **very** important. Try be clever when making your skin!)
```
Comment (TPGPL) [inline wiki/Skinning/Tutorial/en.md]: an idea -> a good idea

try be -> try to be

File: wiki/Skinning/Tutorial/en.md [hunk 8]
```diff
@@ -1,105 +1,136 @@
...
 
 ### What the "new" and "old" style skinning about?
```
Comment (TPGPL) [inline wiki/Skinning/Tutorial/en.md]: no verb in a question alert ❗️

File: wiki/Skinning/Tutorial/en.md [hunk 9]
```diff
@@ -1,105 +1,136 @@
...
-This is an important rule of skinning.
-If you got permission, or user is no longer active and it’s impossible to contact with them, just remember to credit them properly.
+This is an important rule of skinning. If you got permission, or user is no longer active and it’s impossible to contact with them, just remember to credit them properly.
 
 ### What about a skin mashup?
 
-You _could_ do that, but&mdash;even if you do get permission&mdash;please don't share those in the osu! skinning subforums.
-Please **keep those to yourself** or post it on your profile page.
-This will help skinners make their original skins stand out than the remixed/mashup-ed skins.
+You _could_ do that, but&mdash;even if you do get permission&mdash;please don't share those in the osu! skinning subforums. Please **keep those to yourself** or post it on your profile page. This will help skinners make their original skins stand out than the remixed/mashup-ed skins.
```
Comment (TPGPL) [inline wiki/Skinning/Tutorial/en.md]: &mdash;

File: wiki/Skinning/Tutorial/en.md [hunk 10]
```diff
@@ -122,22 +153,23 @@ Those are the three basic categories for skins:
...
 
 - name of a skin that show up in osu!
-- the creator's name (fourm name is prefered)
+- the creator's name (forum name is prefered)
 - skin version (what behavior should osu! use, not your skin's verisoning number)
 - colours
 - osu!mania key, note, and stage behaviors
 - osu!mania image names and paths
 
 ### How do I share?
```
Comment (TPGPL) [inline wiki/Skinning/Tutorial/en.md]: what do you share

File: wiki/Skinning/Tutorial/en.md (hunk 4 above)
Comment (MegaApplePi) [reply wiki/Skinning/Tutorial/en.md]: I'm going to keep it as `that show/game`, I'm not sure if it is common knowledge that "universe" could mean a sandbox where characters/ideas cease to exist 🤷‍♂️

File: wiki/Skinning/Tutorial/en.md (hunk 7 above)
Comment (MegaApplePi) [reply wiki/Skinning/Tutorial/en.md]: I'm going to leave `an idea` as is, not all ideas have to be good to be a good skin 🤷‍♂️

**Pull request merged**